llvm/include/llvm/DebugInfo/Symbolize/Symbolize.h

```diff
@@ -210,7 +208,7 @@ class LLVMSymbolizer {
       ObjectPairForPathArch;
 
   /// Contains parsed binary for each path, or parsing error.
-  std::map<std::string, CachedBinary> BinaryForPath;
+  std::map<std::string, CachedBinary, std::less<>> BinaryForPath;
```

Comment:
The explicit `std::less<>` doesn't seem like it belongs here?

Reply:
`std::less<>` is transparent, counterpart to default map `Compare` being `std::less<std::string>`.
This allows to lookup this map by `StringRef` alongside with `std::string`